frontend/thirdparty/index.js

```diff
@@ -0,0 +1,12 @@
+require('angular');
+require('angular-animate');
+require('angular-bootstrap');
+require('angular-cookies');
+require('angular-resource');
+require('angular-sanitize');
+require('angular-ui-router');
+window.jQuery = require('jquery');
```

Comment:
You'll have to read up on each one of them individually :) but yeah, there's some bloat... we'll drop the jquery dependency eventually.